playground/mocks/data/idp/idx/identify-with-user-verification-universal-link.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
shall we rename to `challenge-authenticator-okta-verify-xxx.json` for those mocks that has `challenge-authenticator` as first remediation?